### PR TITLE
5.3.1 release notes

### PR DESCRIPTION
**Pull request by rrelyea**
5.3.1 nuget.exe and packages was shipped (and blessed) today.
here are the details on the two fixes we took.

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[a316a92](https://github.com/NuGet/docs.microsoft.com-nuget/commits/a316a9229c4a51916d0b1a1e60fb135082bb1126)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.3.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-531releasenotes/docs/release-notes/NuGet-5.3.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1752) | [Details](#user-content-5b18b9cc-1793633c)

<a id="5b18b9cc-1793633c"></a>
### [docs/release-notes/NuGet-5.3.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-531releasenotes/docs/release-notes/NuGet-5.3.md)
 - **Line 26**: **[Suggestion]** `````Link https://docs.microsoft.com/en-us/nuget/nuget-org/deprecate-packages contains locale code en-us. For localizability, remove en-us from links to Microsoft sites. NOTE: This Suggestion will become a Warning on 12/20/2019.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C25%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201910252201251016-1752%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=35osYgC%2FZ5AjjcaLVxPOdNbNzZpCPrc4sXXWhF%2BRuf0%3D&st=2019-10-25T21%3A57%3A36Z&se=2019-11-25T22%3A02%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/release-notes/NuGet-5.3.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ NuGet distribution vehicles:
 | NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
 |:---|:---|:---|
 | [**5.3.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.3](https://visualstudio.microsoft.com/downloads/) | [3.0.100](https://dotnet.microsoft.com/download/dotnet-core/3.0)<sup>1</sup> |
-
+| [**5.3.1**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.3.6](https://visualstudio.microsoft.com/downloads/) | [Future version: 3.0.10x](https://dotnet.microsoft.com/download/dotnet-core/3.0) |
```
Comment (heng-liu) [inline docs/release-notes/NuGet-5.3.md]: Change 3.0.10x to 3.0.101 according to Nick's reply: This will be in 3.0.101 that isn’t out yet.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[d31e692](https://github.com/NuGet/docs.microsoft.com-nuget/commits/d31e69274f27551debb1e578fc73b01d32b26c74)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.3.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-531releasenotes/docs/release-notes/NuGet-5.3.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.3?branch=pr-en-us-1752) | [Details](#user-content-5b18b9cc-5b4e806b)

<a id="5b18b9cc-5b4e806b"></a>
### [docs/release-notes/NuGet-5.3.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-531releasenotes/docs/release-notes/NuGet-5.3.md)
 - **Line 26**: **[Suggestion]** `````Link https://docs.microsoft.com/en-us/nuget/nuget-org/deprecate-packages contains locale code en-us. For localizability, remove en-us from links to Microsoft sites. NOTE: This Suggestion will become a Warning on 12/20/2019.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C25%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201910252207486853-1752%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Fdj8Y%2FzCmyFKMu24NgxFvK6cjmZfmbtaldf5S6%2F5HF8%3D&st=2019-10-25T22%3A03%3A58Z&se=2019-11-25T22%3A08%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by rrelyea** [comment]
@karann-msft - please run with this. i'm fine to have merged and shipped whenever you are.

**Pull request merged**